Applications/Utils/SimpleMeshCreation/createMeshElemPropertiesFromASCRaster.cpp

```diff
@@ -252,19 +241,12 @@ int main (int argc, char* argv[])
 			(*materialIds)[dest_mesh->getElement(dest_perm[k])->getID()] = k;
 		}
 
-		FileIO::Legacy::MeshIO mesh_writer;
-		mesh_writer.setPrecision(12);
-		mesh_writer.setMesh(dest_mesh);
-		mesh_writer.writeToFile(out_mesh_arg.getValue());
+		FileIO::writeMeshToFile(*dest_mesh, out_mesh_arg.getValue());
```

Comment:
You could transform `dest_mesh` into a `unique_ptr`, too, like you did in the file above.